File: apps/institution-api/Models/VwCurrentInstitutionPhone.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ProviderApi.Models
{
    public partial class VwCurrentInstitutionPhone
    {
        public int InstitutionPhoneId { get; set; }
        public int InstitutionId { get; set; }
        public string InstitutionType { get; set; }
        public string InstitutionName { get; set; }
        public string InstitutionCode { get; set; }
        public string PhoneUsage { get; set; }
        public string PhoneType { get; set; }
        public int? ListSequence { get; set; }
        public string CountryCallingCode { get; set; }
        public string PhoneNumber { get; set; }
        public string Extension { get; set; }
        public Guid? PhoneGuid { get; set; }
    }
}

File: apps/planmp/src/libs/common/models/vw-current-institution-phone.ts



export class VwCurrentInstitutionPhone {
    public institutionPhoneId: number = 0;
    public institutionId: number = 0;
    public institutionType: string = '';
    public institutionName: string = '';
    public institutionCode: string = '';
    public phoneUsage: string = '';
    public phoneType: string = '';
    public listSequence: number;
    public countryCallingCode: string = '';
    public phoneNumber: string = '';
    public extension: string = '';
    public phoneGuid: string;

    constructor(init? : Partial<VwCurrentInstitutionPhone>) {
        
        Object.assign(this, init);
    }
}

